mewarpx/examples/capacitive_discharge.py

```diff
@@ -228,10 +210,13 @@
 
 mwxrun.init_run(simulation=sim)
 
-print('Set up simulation with')
-print('  dt = %.3e s' % DT)
-print('  Total time = %.3e s (%i timesteps)' % (TOTAL_TIME, max_steps))
-print('  Diag time = %.3e s (%i timesteps)' % (DIAG_INTERVAL, diag_steps))
+##########################
+# Add direct solver
+##########################
+
+anode_voltage = lambda t: VOLTAGE * np.sin(2.0 * np.pi * FREQ * t)
+# comment line below to use the multigrid solver
```

Comment:
And you should also note you need to uncomment `warpx_potential_hi_z` above I assume?